src/app/recipes/recipes-cards/recipes-cards.component.ts

```diff
@@ -0,0 +1,16 @@
+import { Component } from '@angular/core';
+import { Store } from '@ngrx/store';
+import { Observable } from 'rxjs';
+import { selectRecipes } from 'src/app/store/recipes/recipes.selector';
+import { AppState } from 'src/app/store/store';
+import Recipe from 'src/models/Recipe.model';
```

Comment:
you can put folder path into tsconfig `paths` then you can import it using `@models/` etc